rubin_scheduler/utils/healpy_utils.py

```diff
@@ -398,14 +398,20 @@ def _hp_grow_mask(nside, masked_indx_tuple, grow_size=np.radians(2.0), scale=100
     # get the points that way and save an _hpid2_ra_dec call.
     ra, dec = _hpid2_ra_dec(nside, np.arange(hp.nside2npix(nside))[list(masked_indx_tuple)])
     x, y, z = _xyz_from_ra_dec(ra, dec)
+
+    # convert angular distance to Euclidian distance
```

Comment:
Wouldn't this just be dist = 2*sin(grow_size/2) ?

Reply:
Yes. Yes it is.